js/statuspageApi.js

```diff
@@ -9,7 +11,40 @@ class StatuspageApi {
   }
 
   lastIncident() {
```

Comment:
Maybe this method should be named something more descriptive like "mostRecentMaintenance" or similar